Modules/FindCapistrano.cmake

```diff
@@ -67,7 +95,15 @@ FIND_PACKAGE_HANDLE_STANDARD_ARGS(Capistrano DEFAULT_MSG
   )
 
 IF(CAPISTRANO_FOUND)
-  SET( CAPISTRANO_LIBRARY_DIR ${Capistrano_LIBRARY_DIR} )
-  SET( CAPISTRANO_INCLUDE_DIR ${Capistrano_INCLUDE_DIR} )
-  SET( CAPISTRANO_BINARY_DIR ${Capistrano_BINARY_DIR} )
+  SET(CAPISTRANO_LIBRARY_DIR ${Capistrano_LIBRARY_DIR})
+  SET(CAPISTRANO_INCLUDE_DIR ${Capistrano_INCLUDE_DIR})
+  SET(CAPISTRANO_BINARY_DIR ${Capistrano_BINARY_DIR})
+  SET(CAPISTRANO_BINARY_USBPROBE_NAME "USBprobe${Capistrano_BINARY_FILE_NAME_SUFFIX}")
+  SET(CAPISTRANO_LIBRARY_USBPROBE_NAME "USBprobe${Capistrano_LIBRARY_FILE_NAME_SUFFIX}")
+  SET(CAPISTRANO_BINARY_BMODE_NAME "BmodeUSB${Capistrano_BINARY_FILE_NAME_SUFFIX}")
+  IF(Capistrano_SDK_VERSION STREQUAL "cSDK2018")
+    SET(CAPISTRANO_LIBRARY_BMODE_NAME "BmodeUSB${Capistrano_LIBRARY_FILE_NAME_SUFFIX}")
+  ELSEIF(Capistrano_SDK_VERSION STREQUAL "cSDK2016" OR Capistrano_SDK_VERSION STREQUAL "cSDK2013")
+    SET(CAPISTRANO_LIBRARY_BMODE_NAME "Bmode${Capistrano_LIBRARY_FILE_NAME_SUFFIX}")
+  ENDIF()

```

Comment:
Could you add an else condition with an error or warning message?

Reply:
`set_property(CACHE Capistrano_SDK_VERSION PROPERTY STRINGS ${_csdk_versions})` allows only 3 values of this, and they are completely handled by this if/elseif block. But for future-proofing, I will a cautious else block.